src/libConsensus/ConsensusLeader.h

```diff
@@ -80,16 +84,36 @@ class ConsensusLeader : public ConsensusCommon {
   unsigned int m_commitFailureCounter;
   std::map<unsigned int, std::vector<unsigned char>> m_commitFailureMap;
 
-  // Received responses
-  unsigned int m_responseCounter;
-  std::vector<Response> m_responseDataMap;
-  std::vector<Response> m_responseData;
+  // Tracking data for each consensus subset
+  // TODO: the vectors should be replaced by more space efficient DS
+  struct ConsensusSubset {
+    std::vector<bool> m_commitMap;
+    std::vector<CommitPoint> m_commitPointMap;  // Ordered list of commits of
```

Comment:
It looks the original m_commitPointMap and m_commitPoints still been used. Should we rename the variable names in ConsensusSubset so it can be distinguished from the original one?

Reply:
Well, it’s still the commit maps but local to subset which itself distinguishes it from original.